### PR TITLE
Updated Get Started page for JDG 7

### PR DESCRIPTION
**Pull request by cvanball**
The quickstarts.zip is not updated yet. I want to get this pull request reviewed and accepted when the quickstart.zip is updated including the settings.xml

Thanks in advance,

### CODE REVIEW

**Comment by paulrobinson** [comment]
this is ok to test

**Comment by LightGuard** [comment]
retest this please

**Comment by robpblake** [comment]
retest this please

File: products/datagrid/get-started.adoc [hunk 1]
```diff
@@ -5,77 +5,75 @@
...
+a. For Windows and Mac: Red Hat Developers typically downloads from Oracle’s link: http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html[site].  (Make sure you get the Java *Development* Kit and not just the Java Runtime Environment).
+b. For Red Hat Enterprise Linux: Install OpenJDK by executing the following command:
 +
-`yum install java-1.7.0-openjdk-devel`
+`yum install java-1.8.0-openjdk-devel`
 +
-TIP: You can link:https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6/html/Installation_Guide/Install_OpenJDK_on_Red_Hat_Enterprise_Linux.html[learn more] about installing OpenJDK on RHEL at the customer portal.
 
-=== Install and Configure JBoss Data Grid
+=== Install and Configure JBoss Data Grid 7
```
Comment (paulrobinson) [inline products/datagrid/get-started.adoc]: See: https://www.dropbox.com/s/noco0q6ijp4os56/Screenshot%202016-07-26%2011.39.29.png?dl=0

It looks like the heading is not being set correctly. I've looked at the code on this line and it looks right to me. @LightGuard any ideas?

**Comment by paulrobinson** [comment]
Apart from my above comment, it seems to be rendering the other changes correctly.

**Comment by LightGuard** [comment]
It worked locally for me just fine.

**Comment by LightGuard** [comment]
retest this please

**Comment by cvanball** [comment]
Hi Adela,




Met vriendelijke groet/Kind regards,

Red Hat BV


Cojan van Ballegooijen
Senior Solutions Architect
Building The Entree II
De Entree 238
1101 EE Amsterdam
tel: +31 20 5651225
mob: +31 6 21261996
email: cojan@redhat.com

**Comment by paulrobinson** [comment]
This PR was replaced.

**Pull request closed without merging**